index.js

```diff
@@ -337,7 +337,8 @@ async function sendToLaud(batch, { global, jobs }) {
         // Retry using exponential backoff based on how many retries were already performed
         const nextRetryMs = 2 ** (batch.retriesPerformedSoFar || 0) * 3000 // 2^0 * 3000 = 3000ms, 2^9 * 3000 = 1,536,000ms
         console.error(`Error uploading payload to laudspeaker: ${err}`)
-        console.log(`Enqueued batch ${batch.batchId} for retry in ${Math.round(nextRetryMs / 1000)}s`)
+        console.log(`Enqueued batch ${batch.batchId} of size ${batch.batch.length} for retry in ${Math.round(nextRetryMs / 1000)}s`)
+        console.log(JSON.stringify(payload, null, 2))
```

Comment:
The payload can be pretty big and contain personally identifiable information, we cannot run that code on Cloud. The batch size change is fine though.

If you're struggling to debug this, I suggest running [a hobby deploy](https://posthog.com/docs/self-host), where you could run your own app with improved logging.

Reply:
ok ive gone ahead and removed that console.log statement. Should be good to go now!